docs/modules/ROOT/pages/concepts.adoc

```diff
@@ -13,8 +13,10 @@ For better understanding of the differences among these different types of syste
 [[concepts-dependencies]]
 == Dependencies
 
-The `micrometer-core` module aims to have minimal dependencies. It does not require any third-party (non-Micrometer) dependencies on the classpath at compile time for applications that use Micrometer.
+The `micrometer-core` module aims to have minimal dependencies.
+It does not require any third-party (non-Micrometer) dependencies on the classpath at compile time for applications that use Micrometer, except for https://jspecify.dev/[JSpecify].
```

Comment:
This is an unrelated change I noticed at the time. This could and probably should be made separately and back-ported.